docs/getting-started.md

```diff
@@ -116,4 +116,4 @@ fn main() {
 }
 ```
 
-More information about each component used in the code can be found in [BDK Documentation](https://docs.rs/bdk/0.15.0/bdk/index.html).
\ No newline at end of file
+More information about each component used in the code can be found in [BDK Documentation](https://docs.rs/bdk/0.26.0/bdk/index.html).
```

Comment:
```suggestion
More information about each component used in the code can be found in [BDK Documentation](https://docs.rs/bdk/latest/bdk/index.html).
```

Also there are several other doc link that point to 0.15.0 (lines 47-40)

Reply:
I agree best to point this link to `latest`.

Reply:
Done in dhttps://github.com/bitcoindevkit/bitcoindevkit.org/pull/136/commits/844687b65a8c25d18a70a6f53e04d1ee890f479b